### ISSUE TITLE
Is there some example?

### ISSUE THREAD

**Issue by shunfeichen** [HaoranMiao/streaming-attention#1]
Hi,
   Is there some example? or I can  use espnet  do it ?

**Comment by HaoranMiao** [comment]
Thanks for the question!
You can use espnet. For example, choose one of the [streaming attention modules](https://github.com/HaoranMiao/streaming-attention/blob/master/attentions.py), e.g. MTA in line 319, and add it in the [file](https://github.com/espnet/espnet/blob/master/espnet/nets/pytorch_backend/rnn/attentions.py) for attention implementation in espnet. 
Remember to initialize the parameter of monotonic_bias separately. So you should change some code in the [file](https://github.com/espnet/espnet/blob/master/espnet/nets/pytorch_backend/initialization.py) about initialization in espnet.

**Comment by shunfeichen** [comment]
> Thanks for the question!
> You can use espnet. For example, choose one of the [streaming attention modules](https://github.com/HaoranMiao/streaming-attention/blob/master/attentions.py), e.g. MTA in line 319, and add it in the [file](https://github.com/espnet/espnet/blob/master/espnet/nets/pytorch_backend/rnn/attentions.py) for attention implementation in espnet.
> Remember to initialize the parameter of monotonic_bias separately. So you should change some code in the [file](https://github.com/espnet/espnet/blob/master/espnet/nets/pytorch_backend/initialization.py) about initialization in espnet.

Thank you, I will try it. By the way, can I use it for streaming transformer?

**Comment by HaoranMiao** [comment]
> > Thanks for the question!
> > You can use espnet. For example, choose one of the [streaming attention modules](https://github.com/HaoranMiao/streaming-attention/blob/master/attentions.py), e.g. MTA in line 319, and add it in the [file](https://github.com/espnet/espnet/blob/master/espnet/nets/pytorch_backend/rnn/attentions.py) for attention implementation in espnet.
> > Remember to initialize the parameter of monotonic_bias separately. So you should change some code in the [file](https://github.com/espnet/espnet/blob/master/espnet/nets/pytorch_backend/initialization.py) about initialization in espnet.
> 
> Thank you, I will try it. By the way, can I use it for streaming transformer?

Yes, you can use it as the encoder-decoder attention in each decoder block of Transformer. BTW, you can refer to this [paper](https://arxiv.org/abs/2001.08290) for the streaming transformer.
